torch_dftd/functions/triplets.py

```diff
@@ -38,9 +38,12 @@ def calc_triplets(
     dst = dst[sort_inds]
 
     if shift is None:
-        shift = torch.zeros((src.shape[0], 3), dtype=dtype, device=edge_index.device)
+        edge_indices = torch.arange(src.shape[0], dtype=torch.long, device=edge_index.device)
+        # shift = torch.zeros((src.shape[0], 3), dtype=dtype, device=edge_index.device)
     else:
-        shift = shift[is_larger][sort_inds]
+        edge_indices = torch.arange(shift.shape[0], dtype=torch.long, device=edge_index.device)
+        edge_indices = edge_indices[is_larger][sort_inds]
+        # shift = shift[is_larger][sort_inds]
```

Comment:
It's okay to remove comment, since `shift` is not used now.
you can also merge edge_indices initialization out of if else block.